### DISCUSSION TITLE
Is there a way to exclude an application from blur?

### DISCUSSION THREAD

**Discussion by d3z0ne** [yshui/picom#781] [Q&A] [answered]
I dont want to change the opacity for the application just blur

Is there a way to use something like 
`blur-background-rule = [
    "0:class_g = 'GLava'"
   `]

Maybe there is already an option for it but I just couldnt find it

**Comment by vincentbernat** [comment, marked as answer]
You have `blur-background-exclude`.

**Reply by d3z0ne** [reply]
just wanted to delete this thread cuz i found it myself.
still thx

**Comment by SIMULATAN** [comment]
Going back to the original approach in the question, will there be a `blur-background-rule` option at some point? I want my regular (bigger) applications to have a stronger blur than dunst, which has a almost one colored background with my normal `blur-strength` of `6`
